server/places.go

```diff
@@ -169,6 +169,79 @@ func (s *Server) GetRelatedPlaces(ctx context.Context,
 	return &pb.GetRelatedPlacesResponse{Payload: string(jsonRaw)}, nil
 }
 
+// GetRelatedLocations implements API for Mixer.GetRelatedLocations.
+func (s *Server) GetRelatedLocations(ctx context.Context,
+	in *pb.GetRelatedLocationsRequest) (*pb.GetRelatedLocationsResponse, error) {
+	if in.GetDcid() == "" || len(in.GetStatVarDcids()) == 0 {
+		return nil, fmt.Errorf("missing required arguments")
+	}
+	if !util.CheckValidDCIDs([]string{in.GetDcid()}) {
+		return nil, fmt.Errorf("invalid DCID")
+	}
+
+	withinPlace := in.GetWithinPlace()
+	samePlaceType := in.GetSamePlaceType()
+	isPerCapita := in.GetIsPerCapita()
+	var prefix string
+	if withinPlace == "" {
```

Comment:
wonder if it's better to have a map from <bool, bool, bool> to prefix in here?

Reply:
Good point - done.